perllib/FixMyStreet/Cobrand/Peterborough.pm

```diff
@@ -363,6 +363,7 @@ sub bin_services_for_address {
     my $schedules = $bartec->Features_Schedules_Get($property->{uprn});
     my %schedules = map { $_->{JobName} => $_ } @$schedules;
     my $open_requests = $self->open_service_requests_for_uprn($property->{uprn}, $bartec);
+    $self->{c}->stash->{open_service_requests} = $open_requests;
```

Comment:
This (and disabling) is fine, but would be good for v similar Bromley code to change to match, or this changed to match that (it uses `service.enquiry_open_events`, looks like, and doesn't show them rather than showing them disabled - think showing them disabled for Bromley would be okay).

Reply:
Also, umm, none of this appears to have any tests?